packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -48,17 +43,24 @@ const defaultProps = {
   hideUnit: false,
 };
 
-const interpretationIndicators = {
-  CRITICAL: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMEHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMELOW: <IconCritical className={cx('icon-interpretation')} />,
-  PANICHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  PANICLOW: <IconCritical className={cx('icon-interpretation')} />,
-  VABNORMAL: <IconCritical className={cx('icon-interpretation')} />,
-  POSITIVE: <IconCritical className={cx('icon-interpretation')} />,
-  ABNORMAL: <IconAbnormal className={cx('icon-interpretation')} />,
-  HIGH: <IconHigh className={cx('icon-interpretation')} />,
-  LOW: <IconLow className={cx('icon-interpretation')} />,
+const interpretationIndicatorMap = {
+  critical: <IconCritical className={cx('icon-interpretation')} />,
+  'critical-high': <IconCritical className={cx('icon-interpretation')} />,
+  'critical-low': <IconCritical className={cx('icon-interpretation')} />,
+  positive: <IconCritical className={cx('icon-interpretation')} />,
```

Comment:
These are all using the same critical icon. Can we slim these interpretations options to:

```
critical,
positive,
abnormal,
high,
low
```

Also, in this context, whats the difference between critical-high end vs high and critical-low and low?

Reply:
The critical-high and critical-low have potential to have unique styling based on their level/"priority" being different or alternately presented in additional uses.
Key reason we have them separated here is for accessibility phrases that will be plugging in to be read back to the user as a verbal paragraph of what the visual icons represent, since they have unique clinical concepts.